airavata-api/airavata-client-sdks/airavata-ruby-sdk/README.md

```diff
@@ -0,0 +1,57 @@
+# airavata-ruby-sdk
+
+Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/airavata/ruby/sdk`. To experiment with that code, run `bin/console` for an interactive prompt.
+
+TODO: Delete this and the text above, and describe your gem
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'airavata'
+```
+
+And then execute:
+
+    $ bundle install
+
+Or install it yourself as:
+
+    $ gem install airavata
+
+## Usage
+
+```ruby
+require "thrift"
+require "airavata"
+
+transport = Thrift::BufferedTransport.new(Thrift::Socket.new('localhost', 9930))
```

Comment:
Is it possible to make sever configs to be loaded from a config file and hide transport creation from the user ?

Reply:
Done